code/datums/jobs/job/sectoid.dm

```diff
@@ -0,0 +1,38 @@
+/datum/job/sectoid
+	access = ALL_ANTAGONIST_ACCESS
+	minimal_access = ALL_ANTAGONIST_ACCESS
+	skills_type = /datum/skills/crafty
+	faction = "Sectoids"
+
+
+//Sectoid Grunt
+/datum/job/sectoid/grunt
+	title = "Sectoid Grunt"
```

Comment:
*Might* want to add a Leader/SL role in the way most other ERTs do but that's up to you

Reply:
I might add some kind of "leader" role later on but I don't want to just make a generic leader sectoid